perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -108,6 +225,15 @@ def _convert_value(value, conversion_dict, default_unit=''):
 
 
 def _get_rw(rw_value):
+  """Converting read/randread/write/randwrite to just read/write.
+
+  Args:
+    rw_value: str, 'read'/'randread'/'write'/'randwrite'
```

Comment:
what is the need for single quotes here. just say accepted values are read/randread/write/randwrite